### PR TITLE
Maker funds new wallets

### PR DESCRIPTION
**Pull request by ChewingGlass**
For fresh wallets (0 sol) that are onboarding a new hotspot, fund them with a configurable amount of Sol. This will alleviate onboarding issues where users are unable to claim rewards or perform basic actions.

### CODE REVIEW

**Review by bryzettler — APPROVED** [review]

**Review by ThornM9 — APPROVED** [review]

File: src/config/config.js [hunk 1]
```diff
@@ -9,11 +9,5 @@ module.exports = {
     port: process.env.PGPORT,
     host: '127.0.0.1',
     dialect: 'postgres',
-    dialectOptions: {
```
Comment (ke6jjj) [inline src/config/config.js:12]: You really want to change this at the same time as the PR, and in production?

Comment (ChewingGlass) [reply src/config/config.js:12]: This if for development, not prod?

Comment (ChewingGlass) [reply src/config/config.js:12]: The change shouldn't affect prod

**Comment by shawaj** [comment]
Just posting this here for visibility, but also at https://discord.com/channels/404106811252408320/1116052326692696125/1116737250198294608

> I was thinking about this more, and it feels like the 0.02 SOL thing is not necessarily a good idea as it is in that PR.
>
> Reason I say that is that if SOL goes up to $200 then that's $4 instead of $0.40.
>
> Maybe this should grab an oracle SOL value and work out $0.25 (or whatever value) in current SOL rate instead

**Pull request merged**